src/css/application.css

```diff
@@ -471,19 +471,29 @@ body {
   height: 40vh;
 }
 
-.console__entry,
+.console__entry-active,
 .console__input {
   margin-bottom: 0.5em;
 }
 
-.console__entry {
+.console__entry-active {
   margin-left: 4px;
 }
 
-.console__error {
+.console__error-active {
   color: var(--color-red);
 }
 
+.console__entry-inactive {
+  margin-bottom: 0.5em;
+  margin-left: 4px;
+  color: var(--color-low-contrast-gray);
```

Comment:
I think low-contrast gray is the one we definitely don’t want to use, since the contrast will too low for accessibility.

No strong opinions otherwise, let’s use whatever gray looks best and also passes e.g. [this](https://webaim.org/resources/contrastchecker/)